packages/space-opera/src/components/model_viewer_snippet/components/open_button.ts

```diff
@@ -114,21 +130,35 @@ export class OpenModal extends LitElement {
   }
 
   render() {
-    const exampleLoadableSnippet = `<model-viewer
-  src='https://modelviewer.dev/shared-assets/models/Astronaut.glb'
-  shadow-intensity="1" camera-controls>
-</model-viewer>`;
+    //// Get the model-viewer snippet for the edit snippet modal
+    const editedConfig = {...this.config};
+    applyCameraEdits(editedConfig, this.camera);
+    if (this.gltfUrl && !isObjectUrl(this.gltfUrl)) {
+      editedConfig.src = this.gltfUrl;
+    } else {
+      // Uploaded GLB
+      editedConfig.src = `Change this to your GLB URL`;
+    }
+    if (editedConfig.environmentImage &&
+        isObjectUrl(editedConfig.environmentImage)) {
+      // Uploaded env image
+      editedConfig.environmentImage = `Change this to your HDR URL`;
+    }
+    const exampleLoadableSnippet =
+        renderModelViewer(editedConfig, {}, undefined);
+    ////
```

Comment:
What's up with all the slashes?

Reply:
just trying to bind a comment, will remove them